MANIFEST.in

```diff
@@ -1,17 +1,20 @@
-recursive-include spyder *.pot *.po *.svg *.png *.css *.qss *.c *.cpp *.html *.java *.md *.R *.csv *.pyx *.ipynb *.xml
-recursive-include spyder_breakpoints *.pot *.po *.svg *.png
-recursive-include spyder_profiler *.pot *.po *.svg *.png
-recursive-include spyder_pylint *.pot *.po *.svg *.png
-include spyder/fonts/spyder.ttf
-include spyder/fonts/spyder-charmap.json
-include scripts/*
-include img_src/*.ico
-include img_src/spyder.png
-include img_src/spyder3.png
-include MANIFEST.in
-include README.md
-include LICENSE
-include CHANGELOG.md
-include bootstrap.py
-include runtests.py
-include conftest.py
+recursive-include spyder *.pot *.po *.svg *.png *.css *.qss *.c *.cpp *.html *.java *.md *.R *.csv *.pyx *.ipynb *.xml *.txt
```

Comment:
Do you change line endings in this file? If that's the case, please revert that change and don't do that anymore. It makes my life really hard when merging with master. Also, don't change the order of lines here.

Look, you wanted to make these changes against 3.x, so it's up to you to make things easier to make with master.

Reply:
Done